### PR TITLE
adding steel knuckles

### PR DESCRIPTION
**Pull request by FayFlightee**
#### Summary
Content "adding steel knuckles"

#### Purpose of change

Currently, the knuckles we have are makeshift nail knuckles, somewhat better scrap knuckles, and the uncraftable brass knuckles. So I figured there's no reasons why survivors cannot make steel knuckles with forge equipments, which are better than scrap that it's durable melee. Also added the item to all items pool that have brass knuckles, which, on average, the steel knuckles will be in place at 1/3 chance, and the brass knuckles' spawn rate had been also slightly reduced to make place for steel knuckles spawning,

#### Describe the solution

Simply allow the player characters to forge durable knuckles for unarmed weapon that is better than scrap knuckles. Also allow them to spawn at lower rate than the brass one.

#### Describe alternatives you've considered

Maybe change the scrap knuckles to be tougher one as someone had suggested, but I figured I might as well add an upgrade, since the scrap knuckles are made from hammering crudely into shape, while the steel knuckles is forged from one piece.

#### Testing

Tested the weapon and the recipe, worked fine. Also made sure there's no errors on game startup.

#### Additional context

image

### CODE REVIEW

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json [hunk 1]
```diff
@@ -1,1928 +1,1376 @@
...
-      [ "touristmap", 2 ],
-      [ "brioche", 50 ]
-    ]
+    "subtype": "distribution",
+    "entries": [ { "item": "pants", "prob": 70 } ]
   },
   {
+    "id": "chair_feet",
     "type": "item_group",
-    "id": "pawn",
```
Comment (casswedson) [inline data/json/itemgroups/Locations_MapExtras/locations_commercial.json:197]: This means you deleted some itemgroups (or just the ids) in this file, not sure how many.
The solution that comes to mind and isn't super complicated would be restoring the file to before your changes and do your changes again.
image

**Comment by Shodan14** [comment]
Nice to see the unarmed weapons get a bit more love.

**Comment by FayFlightee** [comment]
casswedson
Not exactly a work in progress as all codes I think are added. Is there anything I still should do?

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json (hunk 1 above)
Comment (FayFlightee) [reply data/json/itemgroups/Locations_MapExtras/locations_commercial.json:197]: Found out about that, fixing it soon

Comment (FayFlightee) [reply data/json/itemgroups/Locations_MapExtras/locations_commercial.json]: fix pushed

**Comment by FayFlightee** [comment]
> Nice to see the unarmed weapons get a bit more love.

Mhm! Unarmed isn't that bad of a choice with proper MAs, or even with just brawling. And after me noticing this not being a thing yet, I figured I might as well make a PR to add it.

**Comment by Tommy-os** [comment]
There could be a spiked steel knuckles variant, an improved version of nail knuckles with some extra cut damage.

**Pull request merged**